renovate_presets/charm.json5

```diff
@@ -15,11 +15,11 @@
   "timezone": "Etc/UTC",
   // Exclude Renovate PRs from auto-generated release notes
   "labels": ["not bug or enhancement"],
-  "enabledManagers": ["poetry", "github-actions", "regex"],
+  "enabledManagers": ["poetry", "pep621", "github-actions", "regex"],
```

Comment:
The manager is pep621, `lockFileMaintenance` from line 11 will make renovate updated the `uv.lock` as well.